interface-definitions/include/firewall/common-rule.xml.i

```diff
@@ -280,6 +280,9 @@
           <format> </format>
           <description>\n\n  Allowed values for TCP flags : SYN ACK FIN RST URG PSH ALL\n  When specifying more than one flag, flags should be comma-separated.\n For example : value of 'SYN,!ACK,!FIN,!RST' will only match packets with\n  the SYN flag set, and the ACK, FIN and RST flags unset</description>
         </valueHelp>
```

Comment:
Please also add valueHelp for each individual flag and also a completionHelper list so tab complete will work :)